pages/doc/wavefront_kubernetes_faq.md

```diff
@@ -7,13 +7,13 @@ permalink: wavefront_kubernetes_faq.html
 summary: Answers about viewing Kubernetes metrics in Tanzu Observability by Wavefront.
 ---
 
-The [Wavefront Collector for Kubernetes on GitHub](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes) enables monitoring Kubernetes clusters and sending metrics to Tanzu Observability. Much of the doc for the project is in the [docs folder on GitHub](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes/tree/main/docs). This page has some special tips and tricks to help you create the user experience you're after.
+The [Wavefront Collector for Kubernetes on GitHub](https://github.com/wavefrontHQ/observability-for-kubernetes) enables monitoring Kubernetes clusters and sending metrics to Tanzu Observability. Much of the doc for the project is in the [docs folder on GitHub](https://github.com/wavefrontHQ/observability-for-kubernetes/tree/main/docs/collector). This page has some special tips and tricks to help you create the user experience you're after.
 
 <!--- Consider including Improve Display Speed with Sampling Option here --->
 
 ## What Is the Wavefront Collector for Kubernetes?
 
-The [Wavefront Collector](https://github.com/wavefrontHQ/wavefront-collector-for-kubernetes) is an agent that you deploy within a Kubernetes cluster as a [DaemonSet](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/). This agent collects metrics about a Kubernetes cluster and sends them to the Wavefront service. You can visualize these metrics using pre-built dashboards that help you monitor your Kubernetes clusters.
+The [Wavefront Collector](https://github.com/wavefrontHQ/observability-for-kubernetes) is an agent that you deploy within a Kubernetes cluster as a [DaemonSet](https://kubernetes.io/docs/collector/concepts/workloads/controllers/daemonset/). This agent collects metrics about a Kubernetes cluster and sends them to the Wavefront service. You can visualize these metrics using pre-built dashboards that help you monitor your Kubernetes clusters.
```

Comment:
https://kubernetes.io/docs/collector/concepts/workloads/controllers/daemonset/ doesn't work

Reply:
Whoops, that should have been changed. Fixed